src/client/pages/Varen/Varen.tsx

```diff
@@ -1,7 +1,79 @@
-import { useAppStateGetter } from '../../hooks/useAppState';
+import { Grid, Icon, Paragraph } from '@amsterdam/design-system-react';
+import { ExternalLinkIcon } from '@amsterdam/design-system-react-icons';
 
-export default function Varen() {
-  const { VAREN } = useAppStateGetter();
-  console.log(VAREN);
-  return 'FAKE';
+import { useVarenThemaData } from './useVarenThemaData.hook';
+import { buttonItems, linkListItems } from './Varen-thema-config';
+import { VarenVergunningFrontend } from '../../../server/services/varen/config-and-types';
+import { Datalist } from '../../components/Datalist/Datalist';
+import { MaButtonLink } from '../../components/MaLink/MaLink';
+import { ThemaTitles } from '../../config/thema';
+import ThemaPagina from '../ThemaPagina/ThemaPagina';
+import ThemaPaginaTable from '../ThemaPagina/ThemaPaginaTable';
+
+const pageContentTop = (
+  <Paragraph>
+    De passagiersvaart in Amsterdam is erg populair bij bezoekers.
+    Rondvaartboten en salonboten zijn een vorm van passagiersvaart. Ook gehuurde
+    boten, met of zonder schipper, vallen onder de passagiersvaart.
+  </Paragraph>
+);
+
+export function Varen() {
+  const {
+    gegevensAanvrager,
+    varenVergunningen,
+    tableConfig,
+    isLoading,
+    isError,
+  } = useVarenThemaData();
+
+  const pageContentTopSecondary = (
+    <>
+      {!!buttonItems &&
+        buttonItems.map(({ to, title }) => (
```

Comment:
Volgens mij kun je ook dit doen toch?
```suggestion
      {buttonItems?.map(({ to, title }) => (
```